packages/engine-server/src/enginev2.ts

```diff
@@ -235,6 +235,10 @@ export class DendronEngineV2 implements DEngine {
     return DendronEngineV2._instance;
   }
 
+  /**
+   * @deprecated see {@link DendronEngineV2.getAllNotes}
+   * For accessing a specific note by id, see {@link DendronEngineV2.getNote}
+   */
   get notes(): NotePropsByIdDict {
```

Comment:
do we need to do this if we are going to be updating the engine interface anyways to `engine.notes.[method]`? might represent unnecessary churn

Reply:
I want to deprecate this first to avoid usages of `this.engine.notes` before we get a clear idea of the engine interface. `engine.notes.method` might not be valid anymore since the business logic lies in the engine vs notes.

Either way, I think its worth preventing others from using the wrong method now.